test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -0,0 +1,37 @@
+Describe "Test-Connection" -Tags "CI" {
+    if ( -not($IsWindows)) {
+         $PSDefaultParameterValues["it:skip"] = $true
+    }
```

Comment:
Please use the pattern https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk